compositions/cluster-k8s/big-blocks/tc-002-da-sync/100-100-50-1000/320-100mib-0ms.toml

```diff
@@ -5,20 +5,44 @@
 [global]
   plan = "celestia"
   case = "002-da-sync"
-  total_instances = 2250
+  total_instances = 2255
```

Comment:
Non-Blocking: should the name of the file also reflect the additional 5 seeds? (_applies to the rest of the files_)

Reply:
ideally yes. Parking into a separate issue